src/Resolver/ObjktResolver.ts

```diff
@@ -137,6 +137,17 @@ export class ObjktResolver {
     return ctx.objktAvailableRedeemablesLoader.load(objkt.id)
   }
 
+  @FieldResolver(returns => Number)
```

Comment:
i think we need `@FieldResolver(returns => Number, { nullable: true })` here as the schema will get the type for `mintedPrice` from this field resolver

Reply:
apparently we don't - ignore!